aws/services/Config/Config.yaml

```
@@ -130,7 +130,7 @@ Resources:
               });
           }
       Handler: index.handler
-      Runtime: nodejs
+      Runtime: nodejs14.x
```

```suggestion
      Runtime: nodejs16.x
```
https://github.com/awslabs/aws-cloudformation-templates/pull/317/files#r883910436